### PR TITLE
contactbot freshening 

### PR DESCRIPTION
**Pull request by technillogue**
- redirect all other requests to tiprat
- towards a fresh contacbot deploy
- bump contactbot syntax
- seemingly necessary bugfixes
- tiprat bugfixes, sylv totally called this one
- drop unused argument for /register

### CODE REVIEW

**Comment by cxloe** [comment]
going to do some basic things to make this more generic for other providers.

**Comment by technillogue** [comment]
@cxloe @i-infra not sure if this is missing stuff

**Comment by technillogue** [comment]
there's probably some other weird bugs hiding but maybe we merge this as is for now?

File: contact/contactbot.py [hunk 1]
```diff
@@ -31,14 +33,28 @@ async def wrapped_command(self: "PayBot", msg: Message) -> str:
...
+        self.api = api.Teli()
         self.payments_manager = PaymentsManager()
         self.routing_manager = RoutingManager()
         self.group_routing_manager = GroupRoutingManager()
         super().__init__(*args)
 
+    async def do_flow(self, message: Message) -> Response:
+        if not message.arg1:
+            # so i'm not sure asking a freeform question like this makes sense
+            # people can just say 'order' or 'status' on its own
```
Comment (cxloe) [inline contact/contactbot.py:47]: a full questionbot flow would be nice to have for UX

File: forest/core.py [hunk 2]
```diff
@@ -740,10 +741,10 @@ async def log_activity(self) -> None:
         runs in the bg as batches to avoid a seperate db query for every message
         used for signup metrics
         """
-        if not self.activity.pool:
-            await self.activity.connect_pg()
+        if not pghelp.pool.pool:
```
Comment (cxloe) [inline forest/core.py:744]: any conflicts here?

**Review by cxloe — APPROVED** [review]
can leave the questionbot bits in as a stub but seems good ?

**Pull request closed without merging**